src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -301,16 +300,24 @@ private void attachCaseByCcdId(
             hasJourneyClassificationForAttachToCase(exceptionRecordData);
         if (classificationValidation.isValid()) {
             if (SUPPLEMENTARY_EVIDENCE_WITH_OCR == classificationValidation.get()) {
-                ServiceConfigItem serviceConfigItem = getServiceConfig(exceptionRecordData);
-                ExceptionRecord exceptionRecord = exceptionRecordValidator.getValidation(exceptionRecordData).get();
-                ccdCaseUpdater.updateCase(
-                    exceptionRecord,
-                    serviceConfigItem,
-                    ignoreWarnings,
-                    idamToken,
-                    userId,
-                    targetCaseCcdRef
-                );
+                Validation<String, String> serviceNameValidationResult =
+                    hasServiceNameInCaseTypeId(exceptionRecordData);
```

Comment:
There's no point validating service name after it's already been validated at an earlier stage of request processing.

Reply:
But still, this is an improvement to the original version

Reply:
Leszek, are you not off today?

Reply:
This is clearer, done using less code and is some step towards removing validation (and handling its failure) from the place in the code that shouldn't be validating things which were already validated.

Reply:
This is clearer, done using less code and is some step towards removing validation (and handling its failure) from the place in the code that shouldn't be validating things which were already validated.

Reply:
Removing or not - extra if/else is not necessary. will certainly make even more clearance if used `.getOrElseThrow` right here. Just a third party suggestion here, not a blocker comment

Reply:
There will be nothing to invoke `getOrElseThrow` on when we remove this duplicated validation